opensearch-operator/pkg/builders/suite_test.go

```diff
@@ -34,12 +45,51 @@ import (
 
 // +kubebuilder:docs-gen:collapse=Imports
 
-/*
-Now, let's go through the code generated.
-*/
+var k8sClient client.Client
+var testEnv *envtest.Environment
+var cancel context.CancelFunc
 
 func TestAPIs(t *testing.T) {
 	RegisterFailHandler(Fail)
 
 	RunSpecs(t, "Builders Suite")
 }
+
+var _ = BeforeSuite(func() {
```

Comment:
what is changed here?

Reply:
I've added the necessary wrapper and boilerplate code to have integration tests with envtest in this package. It's needed as the functions I'm testing here interact with the kubernetes API.